core/storage/beam_job/gae_models.py

```diff
@@ -132,7 +137,8 @@ class BeamJobRunResultModel(base_models.BaseModel):
     Thus, to return the full results you must fetch all models with matching
     job_id values and concatenate their results.
 
-    The IDs of BeamJobRunResultModel are inconsequential, and only used as keys.
+    The IDs of BeamJobRunResultModel are inconsequential, and generated by using
```

Comment:
This is different for dev and prod, right? Do we need to explain that?

Reply:
Nope, only `BeamJobRunModel` will have IDs that are different in Prod and Dev. `BeamJobRunResultModel` will have their IDs be randomly generated and, generally, unused by any services.

Reply:
Ah got it. Thanks!